website/docs/enterprise/getting-started/install-enterprise.mdx

```diff
@@ -113,87 +84,65 @@ In testing, we used the following values:
   --path=./clusters/management \
-  --personal
+  --personal \
   --components-extra image-reflector-controller,image-automation-controller
 ```
 
 </TabItem>
 
-<TabItem value="gitlab" label="GITLAB">
+<TabItem value="gitlab" label="GitLab">
 
 ```bash
 flux bootstrap gitlab \
   --owner=<gitlab username> \
   --repository=fleet-infra \
   --branch=main \
   --path=./clusters/management \
-  --personal
+  --personal \
+  --components-extra image-reflector-controller,image-automation-controller
 ```
 
 </TabItem>
-
-
 </Tabs>
 
-Your private GitHub repo should have a clusters/management folder that includes the manifests Flux needs to operate, and that also generates a key value pair for Flux to access the repo.
-
-* **owner**      - The username (or organization) of the git repository
-* **repository** - Git repository name
-* **branch**     - Git branch (default "main")
-* **path**       - Path relative to the repository root; when specified, the cluster sync will be scoped to this path
-* **personal**   - If set, the owner is assumed to be a repo user
-
-Go [here](https://fluxcd.io/docs/cmd/) for more information about `flux` and the `flux bootstrap` command. 
-
-:::note At this point your Flux management cluster should be running. Take a look at the repository you created earlier.
-:::
-
-### 2. Install a CAPI provider
-
-:::note `clusterctl` versions
+Your private GitHub repo should have a clusters/management folder that includes the manifests Flux needs to operate, and that also generates a key value pair for Flux to access the repo. 
 
-Download a specific version of clusterctl from the [releases page](https://github.com/kubernetes-sigs/cluster-api/releases). We've tested the example templates provided in this guide with `clusterctl` version `1.1.3`. You might need to use a different version, depending on the CAPI provider you plan to use.
-
-:::
+* **owner**: The username (or organization) of the Git repository
+* **repository**: Git repository name
+* **branch**: Git branch (default "main")
+* **path**: Path relative to the repository root; when specified, the cluster sync will be scoped to this path
+* **personal**: If set, the owner is assumed to be a repo user
 
-You must install a CAPI provider to provision Kubernetes clusters. Visit the [Cluster API Providers](../../cluster-management/cluster-api-providers.mdx) page for more details on providers.
+At this point your Flux management cluster should be running. Take a look at the repository you created earlier.
 
-Here we'll continue with our example instructions for CAPD and CAPA.
+### 6. Install CAPA
 
-<Tabs groupId="infrastructure" default>
-<TabItem value="kind" label="CAPD (kind)">
-
-```bash
-# Enable support for `ClusterResourceSet`s for automatically installing CNIs
-export EXP_CLUSTER_RESOURCE_SET=true
+Generally speaking, you do not need to install a CAPI provider to provision Kubernetes clusters using Weave GitOps Enterprise. But for this example with CAPA, you must.
```

Comment:
I would change this sentence as CAPI _is_ currently the only way to provision Kubernetes clusters **using** Weave GitOps Enterprise, there is no _other_ way, if provisioning clusters is what you're after. Of course, Weave GitOps Enterprise does a lot more than provisioning Kubernetes clusters and those features do not require CAPI (for example, you can also view non-CAPI clusters in the UI) .